### PR TITLE
Deleting node without head pointer in linked list

### PR DESCRIPTION
**Pull request by dikshajoshi18**
issue === #659

### CODE REVIEW

File: Linked_list/deletenodewohead.cpp [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+N = 4
+value[] = {10,20,4,30}
+node = 20
+Output: 10 4 30
+Explanation: After deleting 20 from
+the linked list, we have remaining
+nodes as 10, 4 and 30.
+
+
+complexity O(1)
```
Comment (krishna-NIT) [inline Linked_list/deletenodewohead.cpp:111]: Mention Space and Time Complexity

File: Linked_list/deletenodewohead.cpp [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+/*
+Input:
+N = 4
+value[] = {10,20,4,30}
+node = 20
+Output: 10 4 30
+Explanation: After deleting 20 from
+the linked list, we have remaining
+nodes as 10, 4 and 30.
+
```
Comment (krishna-NIT) [inline Linked_list/deletenodewohead.cpp:109]: And one more sample test case

**Comment by dikshajoshi18** [comment]
I did the required changes.
please review it again.

**Comment by krishna-NIT** [comment]
resolve conflict

**Comment by dikshajoshi18** [comment]
can u give any hint I am unable to figure out the issues?

**Comment by dikshajoshi18** [comment]
https://github.com/Algo-Phantoms/Algo-Tree/pull/1870 
I was a little confused so I created fresh or with changes. Please review it.

**Pull request closed without merging**